### ISSUE TITLE
Do not display any other value if required values given

### ISSUE THREAD

**Issue by akroii** [pdir/mobilede-bundle#28]
## Description
Hello,
in germany theres some laws to display car advertisments by Deutsche Umwelthilfe.
One of our costumers say that if the following values not given, the ad should not display:
```
<?php if(

  ($this->ad['fuelConsumption']) ||
  ($this->ad['fuelConsumptionSE']) ||
  ($this->ad['emission_fuel_consumption_co2_emission']) &&
  ($this->ad['specifics_fuel']) &&
  ($this->ad['emission_fuel_consumption_petrol_type']) &&
  ($this->ad['specifics_emission_sticker']) &&
  ($this->ad['specifics_emission_class']) &&
  ($this->ad['emission_fuel_consumption_energy_efficiency_class']) &&
  ($this->ad['emission_fuel_consumption_combined_power_consumption'])
  
  ): ?>
```
Translated:
CO²-Ausstoß (g/km), Kraftstoffart, Benzin Typ, Feinstaubplakette, Schadstoffklasse, Energieeffizienzklasse, „Kombinierter elektrischer Energieverbrauch

These fields should be required in backend to protect from dissuasiveness.

**Comment by MDevster** [comment]
We will add 2 new checkboxes to respond to the requirements of the Deutsche Umwelthilfe and the requirements of the WLTP (Worldwide harmonized Light Duty Test Procedure).

**Comment by MDevster** [comment]
Fixed in 3.3.0
